Description: Fix automatic dependency guessing.
 Debian splits the Twisted package in several modules, and tahoe-lafs depends
 only on some of them. For the package build process to guess the correct Twisted
 package dependency, the egg requires.txt need to be patched.
Author: Julian Taylor <jtaylor.debian@googlemail.com>
Bug-Debian: http://bugs.debian.org/631163

Index: tahoe/src/allmydata_tahoe.egg-info/requires.txt
===================================================================
--- tahoe.orig/src/allmydata_tahoe.egg-info/requires.txt	2011-06-25 13:03:44.532289385 +0200
+++ tahoe/src/allmydata_tahoe.egg-info/requires.txt	2011-06-25 13:05:04.588809558 +0200
@@ -2,10 +2,11 @@
 zfec >= 1.1.0
 simplejson >= 1.4
 zope.interface
-Twisted >= 2.4.0
+Twisted-Core >= 2.4.0
+Twisted-Web >= 2.4.0
 foolscap[secure_connections] >= 0.6.1
 Nevow >= 0.6.0
 pycrypto == 2.0.1, == 2.1.0, >= 2.3
 pyasn1 >= 0.0.8a
 mock
-pycryptopp >= 0.5.20
\ No newline at end of file
+pycryptopp >= 0.5.20
